python/pyspark/sql/functions.py

```diff
@@ -2681,6 +2681,30 @@ def overlay(src, replace, pos, len=-1):
     ))
 
 
+def sentences(str, lang="", country=""):
```

Comment:
no biggie but I would avoid `str` as it shadows the built-in function `str` in Python. Maybe just simply `string`.

Reply:
Ah, right. `str` is not a good name for Python.